CONTRIBUTING.md

```diff
@@ -101,6 +114,14 @@ We recommend having only a single commit for pull request, so if you need to mak
     $ git add <file>
     $ git commit --amend
     $ git push -f
+    
+A more powerful way of modifying commits than `git commit --amend` is with `git rebase -i HEAD~9`. It allows you to join, reorder, change description of past commits and more.
+
+Alternatively, if there are issues with your git history, you can make another branch and push it to the existing PR:
+
+    $ git checkout master -b <attempt2>
```

Comment:
Oh nice, it's obvious what it does when you see it, but I didn't know it's possible.